messages/runtest.md

```diff
@@ -34,6 +34,31 @@ NOTE: The testRunCoverage value (JSON and JUnit result formats) is a percentage
 
   <%= config.bin %> <%= command.id %> --test-level RunLocalTests --output-dir <path to outputdir> --target-org me@my.org
 
+- Run all tests in the org asynchronously:
+  
+  <%= config.bin %> <%= command.id %> --target-org myscratch
+
+- Run all tests synchronously; the command waits to display the test results until all tests finish:
+  
+  <%= config.bin %> <%= command.id %> --synchronous
+
+- Run specific tests using the --test-level flag:
```

Comment:
The original in the DX Developer Guide has
> Use --test-level to run a subset of tests. See [Understanding Deploy](https://developer.salesforce.com/docs/atlas.en-us.250.0.apexcode.meta/apexcode/apex_deploying_ant_deploy.htm) in the Apex Developer Guide for the list of possible test level values.

Can we still keep that link given that it contains useful information?
![Screenshot 2024-06-27 at 10 26 53 AM](https://github.com/salesforcecli/plugin-apex/assets/139700604/adfbce5d-bd7d-4855-84be-b15f14b82930)

Reply:
Thanks for your feedback @daphne-sfdc . I have a question about one of the NoTestRun values. We don't see it in the source -- https://github.com/salesforcecli/plugin-apex/blob/main/src/commands/apex/run/test.ts#L26. Can you confirm that it's a valid value? Thanks!

Reply:
Copying over my response from Slack:

> I reread the docs and it looks like NoTestRun shouldn't be a valid value for a command used to run tests, since you would be asking the command to run nothing.  It looks like it should only be used for whether tests should be run during deployment, and the default is deploying does not run tests.  So you're right that there should only be 3 options for the `--test-level` flag.